jupyterlab_gitplus/__init__.py

```diff
@@ -42,9 +42,15 @@ def _load_jupyter_server_extension(nb_server_app):
     host_pattern = '.*$'
 
     gh_token = nb_server_app.config.get('GitPlus', {}).get('github_token', '')
+    reviewnb_url_template =  nb_server_app.config.get('GitPlus', {}).get(
+        'reviewnb_url_template',
+        'https://app.reviewnb.com/{owner_login}/{repo_name}/commit/{new_commit}'
```

Comment:
The only thing that differs between various ReviewNB installations is the actual endpoint (e.g. `https://app.reviewnb.com/` for managed cloud vs `https://reviewnb.team.affirm.com/` for self-hosted. The rest of the URL format / template always remains the same. 

In light of the above we should just call this an optional config as below -

reviewnb_endpoint = nb_server_app.config.get('GitPlus', {}).get('self_hosted_reviewnb_endpoint', 'https://app.reviewnb.com/')

After this - we can just use this `reviewnb_endpoint` config everywhere else in handlers or utils etc. This would minimise code changes since we'd just be replacing hard coded REVIEWNB_ENDPOINT value with this config value.

Reply:
We'd need some validation / check to make sure it's a proper URL. And we probably need to add a trailing slash if not provided e.g. 

some users would set it as - 

```python
c.GitPlus.self_hosted_reviewnb_endpoint = "https://reviewnb.team.affirm.com"
```

& others might set it as -

```python
c.GitPlus.self_hosted_reviewnb_endpoint = "https://reviewnb.team.affirm.com/"
```

Reply:
Made this change